### PR TITLE
Update archive link spec

### PR DESCRIPTION
**Pull request by jessieay**
* Was not appended with `spec` so was not running
* Stub constant to to so can create fewer records for spec

### CODE REVIEW

File: app/views/proposals/_archive_link.html.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div class="col-sm-2">
-  <%- unless @approved_data.rows.size < @CLOSED_PROPOSAL_LIMIT %>
-    <%= link_to "View the archive", url_for(:controller => 'proposals', :action => 'archive') %>
-  <%- end %>
+  <% if @approved_data.rows.size > @closed_proposal_limit %>
```
Comment (pkarman) [inline app/views/proposals/_archive_link.html.erb]: should this be `if @approved_data.rows.size >= @closed_proposal_limit` instead?

Comment (jessieay) [inline app/views/proposals/_archive_link.html.erb]: we are limiting the results in the controller to  `@closed_proposal_limit` so I think greater than is all we need

previous logic would have showed archive link if there were 10 records, but there would be nothing to more to see in that case

Comment (pkarman) [inline app/views/proposals/_archive_link.html.erb]: failing test?

Comment (pkarman) [inline app/views/proposals/_archive_link.html.erb]: @jessieay the previous commit (72277b495152fe948b253cba94ca14ace815cf40) intentionally biased in favor of showing the link, even if there was nothing behind it. I can't seem to get into pivotal tracker to find the original card. I'm just noticing that this change basically just reverts the previous change.

Comment (jessieay) [inline app/views/proposals/_archive_link.html.erb]: oh interesting, I assume it was not intentional. Do you remember why we did that?

Comment (jessieay) [inline app/views/proposals/_archive_link.html.erb]: failing test is due to constant stubbing not working as I thought it was. working on it!

Comment (pkarman) [inline app/views/proposals/_archive_link.html.erb]: My memory is limited to the commit message, alas.

Comment (jessieay) [inline app/views/proposals/_archive_link.html.erb]: fixed via 8d6f886

**Comment by pkarman** [comment]
@jessieay failures might need manual restart?

**Comment by jessieay** [comment]
@pkarman done!

**Comment by pkarman** [comment]
@jessieay merge conflict alas

**Comment by jessieay** [comment]
@pkarman updated~

**Pull request merged**